src/worker/sandbox/cgroups/pool.go

```diff
@@ -56,32 +59,28 @@ func (pool *CgroupPool) NewCgroup() Cgroup {
 	pool.nextID++
 
 	cg := &CgroupImpl{
-		name: fmt.Sprintf("cg-%d", pool.nextID),
-		pool: pool,
+		name:   fmt.Sprintf("cg-%d", pool.nextID),
+		pool:   pool,
+		logger: common.CgTopLogger,
 	}
+	cg.logger = *cg.logger.With("cgroup pool", cg.pool.Name, "cgroup", cg.name)
```

Comment:
can't you create if from the pools logger so you don't need to specify the pool name again?